projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/intrinsics/c/LLVMCMathsIntrinsics.java

```diff
@@ -69,4 +73,94 @@ public double executeIntrinsic(double value) {
 
     }
 
+    @NodeChild(type = LLVMDoubleNode.class)
+    public abstract static class LLVMRint extends LLVMDoubleIntrinsic {
+
+        @Specialization
+        public double executeIntrinsic(double value) {
+            return Math.rint(value);
+        }
+
+    }
+
+    @NodeChild(type = LLVMDoubleNode.class)
+    public abstract static class LLVMRFloor extends LLVMDoubleIntrinsic {
+
+        @Specialization
+        public double executeIntrinsic(double value) {
+            return Math.floor(value);
+        }
+
+    }
+
+    @NodeChild(type = LLVMDoubleNode.class)
+    public abstract static class LLVMFloor extends LLVMDoubleIntrinsic {
+
+        @Specialization
+        public double executeIntrinsic(double value) {
+            return Math.floor(value);
```

Comment:
You are right, I accidentally checked `Rfloor` in (at least the function is not registered in `LLVMRuntimeIntrinsicFactory`). I'll remove it again.